recipes/newrelic/infrastructure/super-agent/debian.yml

```diff
@@ -390,6 +391,14 @@ install:
             sed -i '/^\s*#\s*agent_type: "newrelic\/io\.opentelemetry\.collector/s/#//' /etc/newrelic-super-agent/config.yaml
           fi
 
+    config_fleet_id:
+      cmds:
+        - |
+          if [ ! -z "{{.NR_CLI_FLEET_ID}}" ] ; then
+            sed -i 's/s*#\s*fleet_id:/fleet_id:/g' /etc/newrelic-super-agent/config.yaml
+            sed -i 's/fleet_id: FLEET_ID_HERE/fleet_id: {{.NR_CLI_FLEET_ID}}/g' /etc/newrelic-super-agent/config.yaml
```

Comment:
Is it correct to hardcode `FLEET_ID_HERE` ? why are we not using a regex here?

Reply:
(same for the rest below)

Reply:
I followed the same pattern used for API_KEY_HERE, where that [example](https://github.com/newrelic/newrelic-super-agent/blob/main/build/examples/super-agent-config-no-agents.yaml) is replaced by the recipe

Reply:
:) got it, thanks!